common/src/main/java/org/terracotta/angela/common/distribution/Distribution102Controller.java

```diff
@@ -335,7 +334,6 @@ public TerracottaManagementServerInstanceProcess startTms(File kitDir, File work
         .command(startTmsCommand(kitDir))
         .directory(workingDir)
         .environment(env)
-        .redirectError(System.err)
```

Comment:
Won't removing `redirectError(System.err)` mean that we mute the error output?

Reply:
Javadox says that by default the error stream is redirected to the output stream.
But can you double check also to be sure we have the same understanding. Thanks!

Reply:
@saurabhagas : oups! my mistake, I misread. I have updated the PR :-)